benchmark/run_phylomizer.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+# usage: run Phylomizer to build phylogenetic trees of individual gene families
+# notes: the official "KMM" protocol is used
+
+set -eu
+source $(dirname "$0")/utils.sh
+args=(
+    gene_fa_dir  # protein sequences in multi-fasta format (*.fa), one file per gene family
+    gene_tree_dir  # output gene trees will be saved to this directory
+    output_fp
+    scripts_dir
+    phylomizer_install_dir
+    phylomizer_config_fp
+    threads
+    stdout
+    stderr
+    verbose
+)
+get_args "$@"
+
+$verbose && echo "Running Phylomizer .."
+
+# set up
+PWD=$(pwd)
+mkdir -p ${working_dir}/phylomizer
+if [ "${phylomizer_config_fp}" == "None" ]
+then
+    phylomizer_config_fp=${working_dir}/phylomizer/config.txt
+    # modified KMM protocol
+    # minimum number of sequences: 10
```

Comment:
No, the default is 3, but apparently, merely 3 taxa are useless for HGT inference. I think we can discuss on this number in the upcoming meeting.